### PR TITLE
there's only so many times i can pass by a section of terrible code before i can't take it anymore so here is your vore rewrite

### PR DESCRIPTION
**Pull request by silicons**
rewrites vore into components and signals because i'm so, so sick and tired of this

cleans it up too because what the fuck

### CODE REVIEW

**Comment by silicons** [comment]
oh does this mean i finally reached rock bottom as per the slippery slope image from 2016

> vore pr

File: code/modules/vore/vore_digest_act.dm [hunk 1]
```diff
@@ -50,7 +48,7 @@
 /obj/item/clothing/accessory/medal/gold/captain/digest_act(...)
 	return FALSE
 /obj/item/clothing/suit/armor/digest_act(...)
-	return FALSE
+	return FALS
```
Comment (Linzolle) [inline code/modules/vore/vore_digest_act.dm:51]: ???

Comment (nik707) [reply code/modules/vore/vore_digest_act.dm:51]: gamer

**Comment by LetterN** [comment]
## it's happening
[]

**Comment by Echopsychotic** [comment]
is this PR still being worked on?

**Pull request closed without merging**